cmd/virtual-kubelet/provider/store.go

```diff
@@ -73,6 +73,7 @@ type InitConfig struct {
 	RemoteKubeConfig     string
 	HomeClusterID        string
 	RemoteClusterID      string
+	GRPCServer           string
```

Comment:
I would not point out the technology we are using, but instead, why do we need this server. In this case, something like LiqoIpamEndpoint would be a lot easier to understand.